### DISCUSSION TITLE
UseFieldArray error typing

### DISCUSSION THREAD

**Discussion by Rinta01** [react-hook-form/react-hook-form#5676] [Q&A]
Having troubles with typing field array errors (#5638 ). Updated useForm hook with generic type as @bluebill1049 suggested, but still having the issue. Any updates on that?

**Comment by bluebill1049** [comment]
<img width="516" alt="Screen Shot 2021-06-18 at 7 06 22 pm" src="https://user-images.githubusercontent.com/10513364/122536818-481d7b80-d068-11eb-8af8-1aaa57f75e3c.png">

if you take a look at the type suggestion, it's array +  index  + object

**Reply by Rinta01** [reply]
How do I access the error.message for the array? I still don't get it

**Reply by bluebill1049** [reply]
`errors.array[0]...`

**Reply by Rinta01** [reply]
But that would be the error of the array item. I need to display the error I set for the array itself (min). Look at the validation scheme

**Reply by bluebill1049** [reply]
Then that's a separate problem, react hook form read your form values type definition and determine the errors object, however, your schema is at run time while you are checking a group of inputs instead return errors associated to the inputs itself.

**Reply by Rinta01** [reply]
I see. But that was the problem i described from the beginning. Maybe I was a little vague on the explanation, apologies. Is there a workaround? This is still valid code, I only see the problem with typing.

**Reply by bluebill1049** [reply]
Thanks for your politeness. 🙏 at the moment, all errors are associated with inputs for TS. I had put up something related to this problem: https://github.com/react-hook-form/react-hook-form/pull/4986 however got rejected by our team member @kotarella1110

**Reply by bluebill1049** [reply]
@kotarella1110 do you think the PR which i proposed can be a potential solution for above?
